README.md

```diff
@@ -39,10 +39,14 @@ $factory = new RestClientFactory(
     'API key',
 );
 
+// The logger is optional
```

Comment:
maybe mention the interface the logger needs to implement in the comment
I know people can find it by following the variable and opening the RestClient class, but it might be nice to mention this as well in the Readme